sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/VectorIndexType.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+/**
+ * Defines the index type of vector index specification in the Azure Cosmos DB service.
+ */
+public enum VectorIndexType {
```

Comment:
Same comment as above, do we need this class to be public?

Reply:
Retaining the getters as they are is essential for consistency. Therefore, even if getPath is not currently utilized, we should keep it here, as per Fabian's suggestion.